stable/traefik/values.yaml

```diff
@@ -61,3 +61,8 @@ accessLogs:
 #  labelSelector:
 rbac:
   enabled: false
+# Enable the /metrics endpoint, for now only supports prometheus
+# set to true to enable metric collection by prometheus
+metrics:
+  enabled: false
+  prometheus.buckets: 0.1,0.3,1.2,5
```

Comment:
Please correct me if im wrong, it appears to have three options, [here](https://github.com/containous/traefik/blob/master/types/types.go#L328)

```
// Metrics provides options to expose and send Traefik metrics to different third party monitoring systems
type Metrics struct {
	Prometheus *Prometheus `description:"Prometheus metrics exporter type"`
	Datadog    *Datadog    `description:"DataDog metrics exporter type"`
	StatsD     *Statsd     `description:"StatsD metrics exporter type"`
}
```